### ISSUE TITLE
--api-class-name option has no default value

### ISSUE THREAD

**Issue by sawadyecma** [acacode/swagger-typescript-api#433]
Hi. I would like to report the problem happened in our team.

The `--api-class-name` option has no default value.

When our team runs:

```sh
swagger-typescript-api -p ./swagger.yaml
```

### expected output file is:

```
...
export declare class Api<SecurityDataType extends unknown> extends HttpClient<SecurityDataType> {
...
```

### but actual output file is : 
```
...
export class undefined<SecurityDataType extends unknown> extends HttpClient<SecurityDataType> {
...
```

It seems to me that the `--api-class-name` should have a default value, for example `Api`.

### Environment

- version: 11.1.0

**Comment by js2me** [comment]
Hi @sawadyecma thanks for this issue! Fixed in `11.1.1`

**Comment by sawadyecma** [comment]
@js2me 
Tnank you for your fixing !
